app/frontend/src/views/Home.vue

```diff
@@ -1,12 +1,149 @@
 <template>
   <v-container class="home">
-    <h2>Welcome to Document Generation Showcase (DGRSC).</h2>
+    <h2 class="text-center">Welcome to Document Generation Showcase (DGRSC).</h2>
```

Comment:
Probably don't need a period at the end of a title

Reply:
Fixed